channel-sdk/src/androidTest/java/com/zello/channel/sdk/SessionTests.kt

```diff
@@ -0,0 +1,391 @@
+package com.zello.channel.sdk
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.zello.channel.sdk.platform.AudioReceiver
+import com.zello.channel.sdk.platform.AudioReceiverEvents
+import com.zello.channel.sdk.platform.AudioSource
+import com.zello.channel.sdk.platform.AudioSourceEvents
+import com.zello.channel.sdk.platform.Decoder
+import com.zello.channel.sdk.platform.DecoderListener
+import com.zello.channel.sdk.platform.Encoder
+import com.zello.channel.sdk.platform.EncoderListener
+import com.zello.channel.sdk.platform.PlayerListener
+import com.zello.channel.sdk.transport.Transport
+import com.zello.channel.sdk.transport.TransportEvents
+import com.zello.channel.sdk.transport.TransportFactory
+import com.zello.channel.sdk.transport.TransportSendAck
+import org.json.JSONObject
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+internal class TestAudioSource: AudioSource {
+	override val level: Int = 0
+
+	override fun prepare(sampleRate: Int, bufferSampleCount: Int, levelMeter: Boolean, noiseSuppression: Boolean, useAGC: Boolean): Boolean {
+		// TODO: Implement prepare(sampleRate:, bufferSampleCount:, levelMeter:, noiseSuppression:, useAGC:)
```

Comment:
Can't tell if this is a leftover TODO or if you really plan to have an implementation for the mock

Reply:
Will clean up.